js/navigation.js

```diff
@@ -42,6 +42,45 @@ tabContainer.addEventListener('click', function (event) {
 
 var formElement = document.querySelector('.search-form');
 
+function recipeGenerator(recipe) {
+  var p = document.createElement('p');
+  p.onclick = function () {
+    window.open(recipe.url);
+  };
```

Comment:
Please use `addEventListener` rather than `.onclick`. Also, can this possibly be an anchor element so you don't need to use JS to navigate?